bin/getInfoRippled.sh

```diff
@@ -0,0 +1,47 @@
+ #!/bin/bash
+
+export TMP_LOC="/tmp/ripple_info"
+if [ ! -d "${TMP_LOC}" ]; then
+       sudo mkdir "${TMP_LOC}"
+fi
+
+rm -f "${TMP_LOC}/*"
+cd ${TMP_LOC}
+echo "$ls"
+cd ~
+
+# Send output from this script to a log file
+## this captures any messages
+## or errors from the script itself
+
+export LOG_FILE="${TMP_LOC}/get_info.log"
+exec 3>&1 1>>"${LOG_FILE}" 2>&1
+
+## Send all stdout files to /tmp
+
+/opt/ripple/bin/rippled server_info > "${TMP_LOC}/server_info.txt"
```

Comment:
Hardcoding the location of the executable and config file makes it hard to test on other systems (Vinnie in particular asked for this to run on CI). Defaulting to these locations is OK, but we should add command-line switches to override them.